pkgs/development/python-modules/python-snappy/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, pythonPackages, fetchFromGitHub, snappy, cffi }:
+
+buildPythonPackage rec {
+  name = "python-snappy";
+  version = "0.5.3";
+
+  src = fetchFromGitHub {
+    owner = "andrix";
+    repo = "python-snappy";
+    rev = version;
+    sha256 = "0hr7vslrrpxgcgsl5i5zlxay2w3075ah2rqvhrcy9y2pxll14r3n";
+  };
+
+  propagatedBuildInputs = [ snappy pythonPackages.cffi cffi ];
```

Comment:
drop pythonPackages

Reply:
It was necessary to do the reverse, I dropped cffi both as import and as propagated build input. Now only uses the python dependency.

Reply:
(or do you prefer to use "with pythonPackages;" at the top?)